testing/percona-xtradb-cluster-galera/APKBUILD

```diff
@@ -0,0 +1,60 @@
+# Contributor: Peter Szalatnay <theotherland@gmail.com>
+# Maintainer: Peter Szalatnay <theotherland@gmail.com>
+pkgname=percona-xtradb-cluster-galera
+pkgver=5.7.16
+_pkgver=5.7.16-27.19
+pkgrel=0
+pkgdesc="Codership Galera library for Percona XtraDB Cluster"
+url="https://www.percona.com/software/mysql-database/percona-xtradb-cluster"
+arch="all"
+license="GPL2"
+depends=""
+depends_dev=""
+makedepends="scons asio-dev check-dev"
+source="https://github.com/percona/galera/archive/pxc_$_pkgver.tar.gz
+        fix-gu_arch.patch
+        fix-gu_alloc.patch
+        "
+
+_builddir="$srcdir/galera-pxc_$_pkgver"
+prepare() {
+        local i
+        cd "$_builddir"
+        for i in $source; do
+                case $i in
+                *.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
+                esac
+        done
+}
```

Comment:
please replace `_builddir` with `builddir` and remove the `prepare()` function to let abuild apply the patches.